### DISCUSSION TITLE
Home Assistant OS 14.0.rc1

### DISCUSSION THREAD

**Discussion by sairon** [home-assistant/operating-system#3683] [Announcements]
## Home Assistant Operating System

* Bump Buildroot to 2024.02.7 (#3659) @sairon
* Remove unnecessary kernel post-install hook for GRUB-based targets (#3672) @sairon
* Reduce timeout for network time synchronization to 15 seconds (#3669) @sairon

## Raspberry Pi

* Add Hailo-8 firmware for Raspberry Pi AI Kit/HAT on RPi 5 (#3680) @sairon
* RaspberryPi: Update kernel to 6.6.51 - stable_20241008 (#3662) @sairon

## Home Assistant Yellow

* Add initial support for Raspberry Pi CM5 on Yellow (#3667) @sairon
* RaspberryPi: Update kernel to 6.6.51 - stable_20241008 (#3662) @sairon

## Home Assistant Green

* Linux: Update kernel to 6.6.62 (#3679) @sairon

## Open Virtual Appliance

* Add iwlwifi-gl firmware needed for Intel BE200 Wi-Fi 7 card (#3668) @antst
* Add kernel config to support external i2c devices (#3642) @xtsrthm
* Add kernel options required for Atheros 12k wirelss adapters (#3640) @agners
* Linux: Update kernel to 6.6.62 (#3679) @sairon

## Generic x86-64

* Add iwlwifi-gl firmware needed for Intel BE200 Wi-Fi 7 card (#3668) @antst
* Add kernel options required for Atheros 12k wirelss adapters (#3640) @agners
* Linux: Update kernel to 6.6.62 (#3679) @sairon

## Hardkernel ODROID

* Fall back to max eMMC clock to fix Kingston eMMC compat with ODROID-N2 (#3665) @sairon
* Linux: Update kernel to 6.6.62 (#3679) @sairon

## ASUS Tinker

* Linux: Update kernel to 6.6.62 (#3679) @sairon

## Khadas VIM Series

* Linux: Update kernel to 6.6.62 (#3679) @sairon

## Generic aarch64

* Add iwlwifi-gl firmware needed for Intel BE200 Wi-Fi 7 card (#3668) @antst
* Linux: Update kernel to 6.6.62 (#3679) @sairon

## Build System

* Add Kconfig option for Supervisor channel (#3618) @agners
* Bump mikepenz/action-junit-report from 4 to 5 (#3658) @dependabot

## Dependencies

* Bump mikepenz/action-junit-report from 4 to 5 (#3658) @dependabot

<hr /><em>This discussion was created from the release <a href='https://github.com/home-assistant/operating-system/releases/tag/14.0.rc1'>Home Assistant OS 14.0.rc1</a>.</em>

**Comment by tjorim** [comment]
Tested on a HA yellow with cm4 and all good (manually pulled it because the version bump action is waiting on deployment approval).

**Comment by Anto79-ops** [comment]
There are no issues on my NUC 12 (x86-64, bare-metal) with USB SkyConnect and USB Aeotec Z-Stick 7.

The system is running normally.

**Comment by roblamoreaux** [comment]
I tried to update to this release on my Raspberry Pi5 with SSD and Hailo M.2 cards and it gets to waiting for Home Assistant CLI to be ready then does nothing. I waited for 15 minutes and power cycled which brought it back up in OS 13.2

When it was sitting there I could SSH into root, but I don't know what to look for to provide debug info.

**Comment by AllardKatan** [comment]
I just did an update of my OS, but it seems the RC2 had not arrived here yet. So I can report it installs and runs just fine in Virtualbox on Linux host. I did notice just before updating that my system has suddenly been jumping up and down in memory use the past month. That wasn't a problem before. I would have to retrace what happened when that started. Somewhere in November. Or is there a known issue in the last release (I was up to date, 13.2 I think was the previous version??). Screengrab of memory use this year attached.
![image](https://github.com/user-attachments/assets/ed028928-ea51-4778-b6b1-b071a2aa14d3)
